app/src/main/res/values/strings.xml

```diff
@@ -10,4 +10,10 @@
     <string name="humid">Humid</string>
     <string name="precip_probability">Precipitation probability</string>
     <string name="text_see_detail">See detail</string>
+    <string name="home_option">Home</string>
+    <string name="search_option">Search</string>
+    <string name="temperature_option">Temperature</string>
+    <string name="wind_speed_option">Wind speed</string>
+    <string name="history_option">History</string>
+    <string name="degree_F_option">ºF</string>
```

Comment:
sao lại để ký tự omega vậy?
https://stackoverflow.com/questions/3312001/degrees-symbol-as-in-degrees-celsius-fahrenheit-in-a-textview